corehq/apps/couch_sql_migration/management/commands/migrate_domain_from_couch_to_sql.py

```diff
@@ -327,6 +328,6 @@ def iter_chunks(model_class, field, domain, chunk_size=5000):
         values=[field],
         load_source='couch_to_sql_migration',
     )
-    rows = (f for f, in rows)  # unpack row (a single field)
-    rows = with_progress_bar(rows, row_count, prefix=model_name, oneline="concise")
+    rows = (f for f, in rows)  # unpack rows (each has a single field)
```

Comment:
I think your clarification is an improvement. And this is more a request for your opinion than a request for change: Do you think writing this as ...
```python
    rows = (f for [f] in rows)
```
... would make it obvious enough not to require the comment at all? (I think it's just that comma that makes it look odd or confusing.)

Reply:
It's funny, using a list for value unpacking is a feature I did not even knew existed until recently. Previously I thought value unpacking only worked with tuples of names, so I always use a tuple rather than a list. I'm not sure if one is better than the other, but I haven't gotten used to the appearance of list unpacking yet...

Reply:
Interesting. My brain is slower to recognise tuples defined without brackets, so 
```python
    rows = (f for (f,) in rows)
```
would also be easier to read, and using a list just makes it a bit more obvious, for me. ... But using a tuple would [be slightly faster](https://stackoverflow.com/questions/68630/are-tuples-more-efficient-than-lists-in-python/22140115) than the list version.

Reply:
My guess would be that sequence unpacking is a syntactical form that can use either list or a tuple _syntax_, but the C runtime does the same thing in either case (and does not allocate the corresponding data structure in either case).

The value of `x` in this example was unintuitive to me at first, but once you think about it it's obvious (and this does not say anything at all).
```py
>>> x = [y, z] = 1, 2
>>> x
(1, 2)
>>> a = (b, c) = 1, 2
>>> a
(1, 2)
>>> a == x
True
```

```sh
$ python -m timeit -s '[y, z] = 1, 2'
20000000 loops, best of 5: 14.5 nsec per loop
$ python -m timeit -s '(y, z) = 1, 2'
20000000 loops, best of 5: 14.4 nsec per loop
```
I believe that difference is within the margin of error.

Reply:
Nice.

Reply:
Oh my, @esoergel! So simple and definitely the "obvious way to do it." I'll make that change in a subsequent PR.